dotnet-vnc/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM gitpod/workspace-full-vnc:latest
+
+RUN wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > microsoft.asc.gpg && \
```

Nit: You could absorb the line below like so:

```suggestion
RUN wget -qO- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/microsoft.asc.gpg && \
```